splice_ck/src/main/java/com/splicemachine/ck/Utils.java

```diff
@@ -0,0 +1,206 @@
+            stringBuilder.append(Colored.boldWhite(String.format(formats.get(i), tabular.headers.get(i))));
+        }
+        stringBuilder.append("\n");
+        int totalLength = lengths.stream().reduce(0, Integer::sum);
+        totalLength += 5 * lengths.size();
+        String separate = new String(new char[totalLength]).replace("\0", "=");
+        stringBuilder.append(Colored.boldWhite(separate)).append("\n");
+        for(Tabular.Row row : tabular.rows) {
+            for(int i = 0; i < row.cols.size(); ++i) {
+                stringBuilder.append(String.format(formats.get(i), row.cols.get(i)));
+            }
+            stringBuilder.append("\n");
+        }
+        return stringBuilder.toString();
+    }
+
+    public static String toString(ExecRow er) throws StandardException {
+        StringJoiner joiner = new StringJoiner(",");
+        for(int i = 1; i <= er.nColumns(); ++i) {
+            joiner.add(er.getColumn(i).toString());
+        }
+        return joiner.toString();
+    }
+
+    public enum SQLType { BOOLEAN, TINYINT, SMALLINT, INT, BIGINT, DOUBLE, REAL, NUMERIC,
+        DECIMAL, CHAR, VARCHAR, REF, TIMESTAMP, DATE, TIME, LONG_VARCHAR, BLOB, CLOB, BIT,
+        VARBIT, LONGVARBIT, XML, LIST, UNKNOWN }
+
+    private static Map<String, SQLType> typeMap;
+    static {
+        typeMap = new HashMap<>();
+        typeMap.put(TypeId.BOOLEAN_NAME, SQLType.BOOLEAN);
+        typeMap.put(TypeId.TINYINT_NAME, SQLType.TINYINT);
+        typeMap.put(TypeId.SMALLINT_NAME, SQLType.SMALLINT);
+        typeMap.put(TypeId.INTEGER_NAME, SQLType.INT);
+        typeMap.put(TypeId.LONGINT_NAME, SQLType.BIGINT);
+        typeMap.put(TypeId.DECIMAL_NAME, SQLType.DECIMAL);
+        typeMap.put(TypeId.NUMERIC_NAME, SQLType.NUMERIC);
+        typeMap.put(TypeId.DOUBLE_NAME, SQLType.DOUBLE);
+        typeMap.put(TypeId.REAL_NAME, SQLType.REAL);
+        typeMap.put(TypeId.CHAR_NAME, SQLType.CHAR);
+        typeMap.put(TypeId.VARCHAR_NAME, SQLType.VARCHAR);
+        typeMap.put(TypeId.REF_NAME, SQLType.REF);
+        typeMap.put(TypeId.LONGVARCHAR_NAME, SQLType.LONG_VARCHAR);
+        typeMap.put(TypeId.BLOB_NAME, SQLType.BLOB);
+        typeMap.put(TypeId.CLOB_NAME, SQLType.CLOB);
+        typeMap.put(TypeId.DATE_NAME, SQLType.DATE);
+        typeMap.put(TypeId.TIME_NAME, SQLType.TIME);
+        typeMap.put(TypeId.TIMESTAMP_NAME, SQLType.TIMESTAMP);
+        typeMap.put(TypeId.BIT_NAME, SQLType.BIT);
+        typeMap.put(TypeId.VARBIT_NAME, SQLType.VARBIT);
+        typeMap.put(TypeId.LONGVARBIT_NAME, SQLType.LONGVARBIT);
+        typeMap.put(TypeId.XML_NAME, SQLType.XML);
+        typeMap.put(TypeId.LIST_NAME, SQLType.LIST);
+    }
+
+    public static SQLType[] toSQLTypeArray(List<String> values) {
+        SQLType[] result = new SQLType[values.size()];
+        int cnt = 0;
+        for(String value : values) {
+            result[cnt++] = typeMap.get(value);
+        }
+        return result;
+    }
+
+    public static class Colored {
```

Comment:
woah these are some sweet util classes! i'm not sure maybe we would want a way to disable coloring for some weird terminals / file forwarding

Reply:
done, check more recent commits, there is a new option: `"-l"` resp. `"--colors"` which enables/disables coloring.

Reply:
The helper message lets the user believe that `-l` prints coloured output whereas it disabled colours on my side.

Reply:
The same goes for verbose. It seems enabled by default and `-v` disables it.
I'd suggest to have colours enabled, verbose disabled, and the respective options to toggle that.

Reply:
Yes! thanks, I was messing around with these parameters locally to test some stuff, I will set them properly.

Reply:
Now the default values are also printed to the output making it easier to know the default behavior.

Reply:
> The helper message lets the user believe that -l prints coloured output whereas it disabled colours on my side.

the default "fallback" values for boolean flags is false by default, I flipped it so now it is more what we expect from setting `-l` or `-v`.